proposals/0253-callable.md

```diff
@@ -504,11 +508,13 @@ call {}
 
 ## Effect on ABI stability
 
-This proposal is about a syntactic sugar and has no ABI breaking changes.
+Adding a new `call` declaration is an additive change to the ABI.
+
+`call` declarations will not be supported when deploying to a Swift 5.0 runtime.

```

Comment:
@jrose-apple: I wonder what else is missing from the ABI stability/resilience sections - could you PTAL?
Our spirit is to "not support `call` declarations when deploying to older runtimes" - does this make sense?

Reply:
That's okay for "ABI stability": it means you are making an additive change to the ABI. "Resilience" (library evolution) is about what the rules are for library authors who want to add, remove, or change `call` declarations from their types without breaking binary compatibility. I suggest modeling the section after the (admittedly out-of-date) docs/LibraryEvolution.rst, particularly the section on [subscripts](https://github.com/apple/swift/blob/master/docs/LibraryEvolution.rst#subscripts).

Reply:
Gotcha, thank you for weighing in! 🙂

Follow-up question: does it make sense to copy the "permitted changes" from the subscripts section, excluding accessor specific-stuff?
That's very vague - in any case, I'll update SE-0253 and cc you.

Reply:
I'd expect the permitted changes to be very similar to what you can do for a regular function, but the section on regular member functions mostly defers to the section on top-level functions, so I picked the subscript one as the model.

Reply:
Gotcha, that makes sense 👍